src/parkapi_sources/converters/opendata_swiss/models.py

```diff
@@ -0,0 +1,201 @@
+        if not self.operationTime.operatingFrom or not self.operationTime.operatingTo:
+            return None
+
+        # OSM 24/7 has no secs in its timeformat and no endtime 00:00, so we replace with 24:00 and remove the secs
+        opening_time: str = f'{self.operationTime.operatingFrom.strftime("%H:%M")}-{self.operationTime.operatingTo.strftime("%H:%M")}'
+        opening_time = '00:00-24:00' if opening_time == '00:00-00:00' else opening_time
+
+        for days_of_week_input in self.operationTime.daysOfWeek:
+            # If the weekday is within Monday to Friday, count to check later for OSM opening hours Mo-Fr
+            if days_of_week_input in list(OpenDataSwissOperationTimeDaysOfWeek)[:5]:
+                check_counter_weekday += 1
+
+        # If it's open all day and number of opening days is 7, then it is OSM - 24/7. No further handling needed in this case.
+        if opening_time == '00:00-24:00' and len(self.operationTime.daysOfWeek) == 7:
+            return '24/7'
+
+        osm_opening_hour: list = []
+        # If the days are Monday to Friday with same opening time, we can summarize it to the Mo-Fr entry,
+        # otherwise we have to set it separately
+        if check_counter_weekday == 5:
+            osm_opening_hour.append(f'Mo-Fr {opening_time}')
+        else:
+            for weekday in list(OpenDataSwissOperationTimeDaysOfWeek)[:5]:
+                if weekday in self.operationTime.daysOfWeek:
+                    osm_opening_hour.append(f'{weekday.to_osm_opening_day_format()} {opening_time}')
+
+        # Weekends are handled separately
+        for weekend_day in [OpenDataSwissOperationTimeDaysOfWeek.SATURDAY, OpenDataSwissOperationTimeDaysOfWeek.SUNDAY]:
+            if weekend_day in self.operationTime.daysOfWeek:
+                osm_opening_hour.append(f'{weekend_day.to_osm_opening_day_format()} {opening_time}')
+
+        return '; '.join(osm_opening_hour)
+
+
+@validataclass
+class OpenDataSwissFeatureInput:
+    id: str = StringValidator()
+    geometry: GeojsonFeatureGeometryInput = DataclassValidator(GeojsonFeatureGeometryInput)
+    properties: OpenDataSwissPropertiesInput = DataclassValidator(OpenDataSwissPropertiesInput)
+
+    def to_static_parking_site_input(self) -> StaticParkingSiteInput:
+        static_parking_site_input = StaticParkingSiteInput(
+            uid=str(self.id),
+            name=self.properties.displayName,
+            operator_name=self.properties.operator,
+            lat=self.geometry.coordinates[1],
+            lon=self.geometry.coordinates[0],
+            static_data_updated_at=datetime.now(tz=timezone.utc),
+            has_realtime_data=False,
+        )
+
+        if self.properties.parkingFacilityCategory == PurposeType.CAR.value:
+            static_parking_site_input.purpose = PurposeType.CAR
+            static_parking_site_input.type = ParkingSiteType.CAR_PARK
+        elif self.properties.parkingFacilityCategory == PurposeType.BIKE.value:
```

Comment:
I wonder why you check for BIKE at all?

Reply:
There is an attribute ```bikeFacilityTraits``` that is always an empty list. In case of future when the portal includes "BIKE" in ```parkingFacilityCategory```. Plus we also pull the data from the portal, and the ```Änderungsdatum``` on the [portal ](https://opentransportdata.swiss/de/dataset/parking-facilities) changes, which I guess they update the data once in a while.